site/app/libraries/NotificationFactory.php

```diff
@@ -221,8 +221,9 @@ public function sendNotifications(array $notifications): void {
     /**
      * prepare array of Email objects as param array
      * @param array $emails
+     * @param bool $forceSecondary
      */
-    public function sendEmails(array $emails): void {
+    public function sendEmails(array $emails, bool $forceSecondary = false): void {
```

Comment:
variables should be `snake_case` (`$force_secondary`), please fix throughout this PR.